src/js/components/Tabs/Tabs.js

```diff
@@ -49,33 +44,7 @@ const Tabs = forwardRef(
     delete rest.onActive;
     /* eslint-enable no-param-reassign */
 
-    let activeContent;
-    let activeTitle;
-    const tabs = Children.map(
-      children,
-      (tab, index) => {
-        if (!tab) return undefined;
-
-        const tabProps = tab.props || {};
-
-        const isTabActive = index === activeIndex;
-
-        if (isTabActive) {
-          activeContent = tabProps.children;
-          if (typeof tabProps.title === 'string') {
-            activeTitle = tabProps.title;
-          } else {
-            activeTitle = index + 1;
-          }
-        }
-
-        return cloneElement(tab, {
-          active: isTabActive,
-          onActivate: () => activateTab(index),
-        });
-      },
-      this,
-    );
+    const [activeTitle, setActiveTitle] = useState();
```

Comment:
This seems disconnected from its usage. Recommend either placing together with together rest of state declarations, or directly ahead of its usage.

Reply:
Agreed, will group it with the other declarations.